CONTRIBUTING.md

```diff
@@ -129,40 +129,49 @@ It is used on bare systems to bootstrap the Swift toolchain (including SwiftPM),
 The bootstrap script requires having [CMake](https://cmake.org/) and [Ninja](https://ninja-build.org/) installed.
 Please refer to the [_Get Started_ guide](https://github.com/apple/swift/blob/main/docs/HowToGuides/GettingStarted.md#installing-dependencies) on the Swift project repository for installation instructions.
 
-1. Clone [llbuild](https://github.com/apple/swift-llbuild) beside the SwiftPM directory.
-
-```bash
-$> git clone https://github.com/apple/swift-llbuild llbuild
-```
-
-Note: Make sure the directory for llbuild is called "llbuild" and not
-    "swift-llbuild".
-
-2. Clone [Yams](https://github.com/jpsim/yams) beside the SwiftPM directory.
-
-```bash
-$> git clone https://github.com/jpsim/yams
-```
-
-3. Clone [swift-driver](https://github.com/apple/swift-driver) beside the SwiftPM directory.
-
-```bash
-$> git clone https://github.com/apple/swift-driver
-```
-
-4. Clone [swift-argument-parser](https://github.com/apple/swift-argument-parser) beside the SwiftPM directory and check out tag with the [latest version](https://github.com/apple/swift-argument-parser/tags).
-
-For example, if the latest tag is 0.3.1:
-```bash
-$> git clone https://github.com/apple/swift-argument-parser --branch 0.3.1
-```
-
-5. Clone [swift-crypto](https://github.com/apple/swift-crypto) beside the SwiftPM directory and check out tag with the [latest version](https://github.com/apple/swift-crypto/tags).
-
-For example, if the latest tag is 1.1.3:
-```bash
-$> git clone https://github.com/apple/swift-crypto --branch 1.1.3
-```
+Clone the following repositories beside the SwiftPM directory:
+
+1. [swift-driver]
```

Comment:
Perhaps it's just me, but I've always found it easier to think of these dependencies "from the bottom up", i.e. starting with those that don't have any other dependencies, and they working one's way up the layers.  So it's a little odd to see `swift-driver` before `swift-tools-support-core`.  So I would suggest going in dependency order, starting with those that don't have any dependencies and finishing with `swift-package-manager`.

Reply:
Yes this makes sense, and is also consistent with the order they're built in `bootstrap`. I ordered them "from the bottom up" in 2c7df484d0ad9092fa56ad99cb8fdd05a8284a00.

Reply:
> finishing with `swift-package-manager`

Cloning SwiftPM is already mentioned in line 19 so I didn't repeat it in the bootstrap instructions.

Reply:
Thanks!